scripts/generate-engine-mock.sh

```diff
@@ -25,8 +25,8 @@ fi
 
 echo "generate dataflow engine mock code..."
 
-"$MOCKGEN" -package mock github.com/pingcap/tiflow/engine/pkg/meta/metaclient KVClient \
-	>engine/pkg/meta/kvclient/mock/client_mock.go
+"$MOCKGEN" -package mock github.com/pingcap/tiflow/engine/pkg/meta/common KVClient \
```

Comment:
Whether the script should be placed inside the package? e.g:https://github.com/pingcap/tiflow/blob/master/cdc/redo/writer/writer.go#L41

Reply:
Seems that `mockery` and `gomock` are tools for easy mocking. 
May be we can do it in other pr if we really need it.